src/pokete/classes/difficulty_ui.py

```
@@ -43,6 +43,16 @@ def _get_insight_text(self):
             se.Text(f" [{sparkline}]", esccode=Color.thicc) + se.Text("\n\n")
         )
 
+        # Style Resonance Block
+        text += se.Text("Style Resonance:\n", esccode=Color.underlined)
+        for label, val in zip([" - Flawless:", " - Clutch:  ", " - Mastery: "],
+                              [dm.flawless_ema, dm.clutch_ema, dm.mastery_ema]):
```

Ruff (0.14.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)